traitsui/testing/tester/qt4/implementation/list_editor.py

```diff
@@ -8,14 +8,90 @@
 #
 #  Thanks for using Enthought open source!
 #
-from traitsui.testing.tester import locator
+from traitsui.testing.tester import command, locator
 from traitsui.testing.tester.registry_helper import register_nested_ui_solvers
-
+from traitsui.testing.tester.qt4 import helpers
 from traitsui.qt4.list_editor import (
     CustomEditor,
+    NotebookEditor,
 )
 
 
+class _IndexedNotebookEditor:
```

Comment:
This class and `_IndexedCustomEditor` will both be able to be refactored using `_BaseSourceWithLocation` base class once pr #1191 is merged.  This will be done in a later PR